src/metrikker/iatjenester.ts

```diff
@@ -45,21 +59,23 @@ const getIaTjenesterMetrikkerUrl = () => {
 
 const iaTjenesterMetrikkerAPI = `${getIaTjenesterMetrikkerUrl()}/innlogget/mottatt-iatjeneste`;
 
-function byggIaTjenesteMottattMetrikk(nåværendeOrgnr: string | undefined) {
+function byggIaTjenesteMottattMetrikk(
+    nåværendeOrgnr: string | undefined,
+    kilde: string | undefined = IaTjenesteKilde.SYKEFRAVÆRSSTATISTIKK
```

Comment:
for meg er det litt rart å skulle tillate `undefined` som verdi, og samtidig ha en default-verdi. 

Er det i praksis en forskjell på disse to kallene?
`byggIaTjenesteMottattMetrikk("00000000", undefined)` vs `byggIaTjenesteMottattMetrikk("00000000")`